src/main/scripts/install.sh

```diff
@@ -14,16 +14,50 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-resourceGroupName=$1
-aksClusterName=$2
+wait_deployment_complete() {
+    deploymentName=$1
+    namespaceName=$2
+    logFile=$3
+
+    kubectl get deployment ${deploymentName} -n ${namespaceName}
+    while [ $? -ne 0 ]
+    do
+        echo "Wait until the deployment ${deploymentName} created..." >> $logFile
+        sleep 5
```

Comment:
Are we sure we don't want to have a MAX_RETRIES constant?

Reply:
If deployment is not created, the deployment of the offer won't work. That's why an infinite loop is used instead of a loop with MAX_RETRIES.